src/utils/dfx/token/extMethods.ts

```diff
@@ -26,15 +28,16 @@ const send = async (
 
   const transferResult = await actor.transfer(data);
 
-  if ('ok' in transferResult) return transferResult.ok;
+  if ('ok' in transferResult) return { amount: transferResult.ok };
```

Comment:
is this ok? does `ok` contain the transferred amount ?

Reply:
It was that way on methods.ts